### DISCUSSION TITLE
Accessing the margin of an element in scipt

### DISCUSSION THREAD

**Discussion by Miurath** [eclipse-birt/birt#976] [Q&A] [answered]
As stated, I would like to change the margin of an element, lets say a text-element in the script. How would i do this?
First idea was to just access it like i can do with the master page margin but this doesn't work.

As i see it every element has a StyleHandle. But how would i access that in the script and change it?

**Comment by hvbtup** [comment, marked as answer]
You can change the margin in the onCreate event (at least this works for the PDF emitter, haven't tried with other emitters).
e.g.

    this.getStyle().marginTop = "20pt";

(untested, but I did this in a similar way in several reports).
